osu.Game/Overlays/SettingsToolboxGroup.cs

```diff
@@ -23,46 +22,66 @@ namespace osu.Game.Overlays
 {
     public class SettingsToolboxGroup : Container, IExpandable
     {
+        private readonly string title;
         public const int CONTAINER_WIDTH = 270;
 
         private const float transition_duration = 250;
-        private const int border_thickness = 2;
         private const int header_height = 30;
         private const int corner_radius = 5;
 
-        private const float fade_duration = 800;
-        private const float inactive_alpha = 0.5f;
-
         private readonly Cached headerTextVisibilityCache = new Cached();
 
-        private readonly FillFlowContainer content;
+        protected override Container<Drawable> Content => content;
+
+        private readonly FillFlowContainer content = new FillFlowContainer
+        {
+            Name = @"Content",
+            Origin = Anchor.TopCentre,
+            Anchor = Anchor.TopCentre,
+            Direction = FillDirection.Vertical,
+            RelativeSizeAxes = Axes.X,
+            AutoSizeDuration = transition_duration,
+            AutoSizeEasing = Easing.OutQuint,
+            AutoSizeAxes = Axes.Y,
+            Padding = new MarginPadding { Horizontal = 10, Top = 5, Bottom = 10 },
+            Spacing = new Vector2(0, 15),
+        };
 
         public BindableBool Expanded { get; } = new BindableBool(true);
 
-        private readonly OsuSpriteText headerText;
+        private OsuSpriteText headerText = null!;
 
-        private readonly Container headerContent;
+        private Container headerContent = null!;
+
+        private Box background = null!;
+
+        private IconButton expandButton = null!;
 
         /// <summary>
         /// Create a new instance.
         /// </summary>
         /// <param name="title">The title to be displayed in the header of this group.</param>
         public SettingsToolboxGroup(string title)
         {
+            this.title = title;
+
             AutoSizeAxes = Axes.Y;
             Width = CONTAINER_WIDTH;
             Masking = true;
+        }
+
+        [BackgroundDependencyLoader(true)]
+        private void load(OverlayColourProvider? colourProvider)
+        {
             CornerRadius = corner_radius;
-            BorderColour = Color4.Black;
-            BorderThickness = border_thickness;
 
             InternalChildren = new Drawable[]
             {
-                new Box
+                background = new Box
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Colour = Color4.Black,
-                    Alpha = 0.5f,
+                    Alpha = 0,
```

Comment:
I find it weird for the controls to essentially "float" in space like this. Think it looks better at 0.1 alpha.

Reply:
Also the fully faded in background clashes with the slider background:
![image](https://user-images.githubusercontent.com/1329837/195797160-a70e68cc-3865-4cce-8999-7b78b454c69a.png)

Reply:
I've changed it to 10%, look fine.

I've fixed the colour clash by changing the background tint. Note that this changes how the editor displays on hover, but I don't think it looks bad at all.